src/proc/motion-transform.cpp

```diff
@@ -230,17 +239,18 @@ namespace librealsense
         if (source[0] == 1)
         {
             _target_stream = RS2_STREAM_ACCEL;
+            _accel_converter->convert( dest, source );
         }
         else if (source[0] == 2)
         {
             _target_stream = RS2_STREAM_GYRO;
+            _converter->convert( dest, source );
```

Comment:
So `converter` is `_gyro_converter` actually?

Reply:
For a single stream type there is only a single `converter`, however mipi  stream have alternating accel/gyro, so we need 2 different scale factors (converters)

Reply:
Added comment